### DISCUSSION TITLE
Best ASUS Motherboards settings

### DISCUSSION THREAD

**Discussion by InvisibleKnight** [Rem0o/FanControl.Releases#2057] [Q&A]
Hello everyone, I'm creating this topic on order to solve my issues and also try to make sense of what are the best BIOS settings for ASUS motherboards. [There ](https://github.com/Rem0o/FanControl.Releases/issues/325) are [dozen](https://github.com/Rem0o/FanControl.Releases/issues/732) of [us](https://github.com/Rem0o/FanControl.Releases/discussions/1782)! I feel it would be important to mention some information on the setup when sharing your experience. 

In my case:
- Motherboard: ASUS Zenith Extreme Alpha II 
- Sensor chipset: Nuvoton NCT6798D
- PWM Fans connected on CHA Fan 1 and CHA Fan 2 headers
- I removed VRMs Fan and heatsink
- [AsusWMI ](https://github.com/Mourdraug/FanControl.AsusWMI)Plugin installed but doesn't initialize (no sensors found)

I was having countless issues with FanXpert/Ai Suite (specifically my fan curves would stop working during long idle periods and the temperatures would skyrocket for whatever reason). Long story short I finally installed FanControl and got a new round of issues to solve in hope to reach a definitive configuration. Something is still fighting FanControl to control the fans. 

Current Issues:
- FanControl is intermittently fighting to gain control over both CHA Fan 1 and 2. After further investigation I have random RPM ramp ups on all headers (even unplugged ones), image attached: neither the fan curves nor BIOS settings justify this behavior, it doesn't seem triggered by temperatures either. 
![2023-09-29_11h15_53](https://github.com/Rem0o/FanControl.Releases/assets/46795287/d65a3acf-3358-42ac-a938-b8679914db6f)
After further investigation I found [this ](https://forums.tomshardware.com/threads/case-fans-spin-up-without-temp-spike.3752387/)and [this](https://rog-forum.asus.com/t5/zenith-extreme-x399-e/cpu-fans-randomly-ramping-up/td-p/874678), which seem to indicate a bug in the management of the fans themselves. Not sure how to solve it.

- Cannot control or monitor Chipset/VRM fan, it probably will never be possible as Rem0o has expressed his will to keep the software as generalist as possible without going into manufacturer specific shenanigans.

- (minor) Some very inconsistent temps values between Libre Hardware Monitor and AI Suite.

Is there anyway to check which program is trying to take control on the fans at any given time? Just to make sure it is not software related but BIOS related.

What I tried so far:
- I disabled Ai Suite from starting on logon because I thought it was conflicting with FanControl. Probably not the case as I'm facing the same issues with or without it.
Specifically I killed startup applications and the following services: ARMOURY CRATE Service, ASUS AURA SYNC lighting service, ASUS Com Service, ASUS HM Corn Service, ASUS System Control Service, AsusFanControlService, AsusROGLSLService Download, ROGLSLoader.

- I had to use CHA Fan 1 and CHA Fan 2 headers because CPU headers were too restrictive in their setting (no Stop Fan options, can't change reference sensor, etc.).

- Following some advice given by the community [I setup CHA Fan 2 in DC mode](https://github.com/Rem0o/FanControl.Releases/issues/1977#issuecomment-1684568301) (it is a PWM fan) that allows me to enable Stop Fan (0%). By doing so I was able to turn off the fan. I couldn't do the same with CHA Fan 1, I had to set it to PWM. I have no idea why. Both PWM and DC modes need to set a curve so I did so, I suspect these are the settings FanControl is fighting with but I don't believe it's possible to [turn off "smart" controls](https://github.com/Rem0o/FanControl.Releases/issues/732#issuecomment-1116737846) and retain "turn off fans" function.

**Comment by solodolo-42** [comment]
Have you disabled the Q-Fan configuration in the asus bios settings?

**Reply by InvisibleKnight** [reply]
I did for CHA_FAN1 header, nothing really changed. CHA Fan 2 header is in DC mode to be able to enable "Stop Fan"

**Comment by mcantar** [comment]
You may be able to resolve this using HWiNFO and it's respective plugin

**Reply by mcantar** [reply]
I posted too soon, sorry. I would completely uninstall whatever ASUS bloatware you have - Armoury Crate, AIwhatever, FanXpert, etc - then install HWiNFO and the appropriate plugin to bring its reported values over to FanControl. 

The former, along with making sure BIOS is set to "allow software to control" if necessary, will make sure nothing is interfering. The latter *should* let you monitor and control VRM fan and so on
